_inc/client/traffic/site-stats.jsx

```diff
@@ -85,89 +86,95 @@ export const SiteStats = moduleSettingsForm(
 				unavailableInDevMode = this.props.isUnavailableInDevMode( 'stats' ),
 				siteRoles = this.props.getSiteRoles();
 			return (
-				<FoldableCard
-					header={ __( 'Site stats' ) }
-					clickableHeader={ true }
-					className={ classNames( 'jp-foldable-settings-standalone', { 'jp-foldable-settings-disable': unavailableInDevMode } ) }
+				<SettingsCard
```

Comment:
Visual unity. It didn't quite fit the patterns we used everywhere else. Now it fits in a bit better I think.